lib/internal/streams/readable.js

```diff
@@ -316,6 +316,20 @@ function Readable(options) {
   if (!(this instanceof Readable))
     return new Readable(options);
 
+  this._events ??= {
```

Comment:
I'm wondering how common these events are relative to each other and if we should allocate a subset

Reply:
Fixed.